linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/force_opensc_card_drivers/rule.yml

```diff
@@ -1,11 +1,11 @@
 documentation_complete: true
 
-prodtype: fedora,ol7,ol8,rhel7,rhel8,rhv4
+prodtype: fedora,ol7,ol8,rhel7,rhel8,rhel9,rhv4
 
 title: 'Force opensc To Use Defined Smart Card Driver'
 
 description: |-
-    The OpenSC smart card tool can auto-detect smart card drivers; however by
+    The OpenSC smart card middleware can auto-detect smart card drivers; however by
     forcing the smart card driver in use by your organization, opensc will no longer
     autodetect or use other drivers unless specified. This helps to prevent
     users from using unauthorized smart cards. The default smart card driver for this
```

Comment:
on the following line, you have also the `/etc/opensc-<i>ARCH</i>.conf`, which is already fixed in the previous rule. Please, change it also here. I just checked and on normal x86_64 it should work on since RHEL7.